src/Documenter.jl

```diff
@@ -316,6 +317,18 @@ documentation. By default this value is set to `"master"`.
 **`devurl`** the folder that in-development version of the docs will be deployed.
 Defaults to `"dev"`.
 
+**`selector`** determines content and order of the resulting version selector in
+the generated html. The following entries are valied in the `selector` vector:
+ - `"v#"`: includes links to the latest documentation for each major release cycle
+   (i.e. `v1.1`, `v2.0`).
+ - `"v#.#"`: includes links to the latest documentation for each minor release cycle
+   (i.e. `v1.0`, `v1.1`, `v2.0`).
+ - `"v#.#.#"`: includes links to all released versions.
+ - `"v^"`: includes a link to the maximum version.
```

Comment:
What does it put in the selector though? Or can you just use this in a pair?

Reply:
It will put `vX.Y` for latest `X` and `Y`, and symlink to `vX.Y.Z` for latest `Z`.